src/jemalloc.c

```diff
@@ -1482,8 +1482,8 @@ malloc_conf_init_helper(sc_data_t *sc_data, unsigned bin_shard_sizes[SC_NBINS],
 			    "hpa_sec_bytes_after_flush", PAGE, 0,
 			    CONF_CHECK_MIN, CONF_DONT_CHECK_MAX, true);
 			CONF_HANDLE_SIZE_T(opt_hpa_sec_opts.batch_fill_extra,
-			    "hpa_sec_batch_fill_extra", PAGE, 0, CONF_CHECK_MIN,
```

Comment:
This was unfortunate. Did we have flush stats for the hpa?

Reply:
Yes, but I wasn't looking at them with custom malloc_confs after the interface changes.

This only triggers if you try non-default settings, which I didn't in perf-sensitive situations until recently (at which point, the programs failed to boot in a reasonable amount of time).